.github/workflows/OCV-Contrib-PR-3.4-macOS-ARM64.yaml

```diff
@@ -0,0 +1,366 @@
+  GTEST_FILTER_STRING: '-tracking_GOTURN.GOTURN/*:Objdetect_QRCode_Close.regression/0:DenseOpticalFlow_PCAFlow.ReferenceAccuracy:OCL_SuperResolution.BTVL1:SuperResolution.BTVL1:OCL_OCL_Video/Mog2_Update.Accuracy/0:OCL_OCL_Video/Mog2_Update.Accuracy/1:OCL_OCL_Video/Mog2_Update.Accuracy/2:OCL_OCL_Video/Mog2_Update.Accuracy/3:OCL_OCL_Video/Mog2_Update.Accuracy/4:OCL_OCL_Video/Mog2_Update.Accuracy/5:OCL_OCL_Video/Mog2_Update.Accuracy/6:OCL_OCL_Video/Mog2_Update.Accuracy/7:OCL_OCL_Video/Mog2_getBackgroundImage.Accuracy/0:OCL_OCL_Video/Mog2_getBackgroundImage.Accuracy/1:OCL_OCL_Video/Mog2_getBackgroundImage.Accuracy/2:OCL_OCL_Video/Mog2_getBackgroundImage.Accuracy/3:OCL_Video/PyrLKOpticalFlow.Mat/0:OCL_Video/PyrLKOpticalFlow.Mat/1:OCL_Video/PyrLKOpticalFlow.Mat/2:OCL_Video/PyrLKOpticalFlow.Mat/3:OCL_Video/PyrLKOpticalFlow.Mat/4:OCL_Video/PyrLKOpticalFlow.Mat/5:OCL_ResizeFixture_Resize.Resize/26:OCL_ResizeFixture_Resize.Resize/27:OCL_ResizeFixture_Resize.Resize/42:OCL_ResizeFixture_Resize.Resize/43:OCL_ResizeFixture_Resize.Resize/74:OCL_ResizeFixture_Resize.Resize/75:OCL_ResizeFixture_Resize.Resize/90:OCL_ResizeFixture_Resize.Resize/91:Perf_Objdetect_QRCode.detect/2:Perf_Objdetect_QRCode_Multi.detectMulti/0:Perf_Objdetect_QRCode_Multi.detectMulti/1:Perf_Objdetect_QRCode_Multi.detectMulti/3:Perf_Objdetect_QRCode_Multi.detectMulti/4:Perf_Objdetect_QRCode_Multi.detectMulti/5:Perf_Objdetect_QRCode_Multi.detectMulti/6:Perf_Objdetect_QRCode_Multi.decodeMulti/0:Perf_Objdetect_QRCode_Multi.decodeMulti/1:Perf_Objdetect_QRCode_Multi.decodeMulti/3:Perf_Objdetect_QRCode_Multi.decodeMulti/5:Perf_Objdetect_QRCode_Multi.decodeMulti/6:OCL_KNN_Apply_KNN.KNN/*:OCL_KNN_GetBackgroundImage_KNN.KNN/0:OCL_KNN_GetBackgroundImage_KNN.KNN/1:OCL_MOG2_Apply_Mog2.Mog2/*:OCL_MOG2_GetBackgroundImage_Mog2.Mog2/0:OCL_MOG2_GetBackgroundImage_Mog2.Mog2/1:OCL_PyrLKOpticalFlowFixture_PyrLKOpticalFlow.PyrLKOpticalFlow/*:KNN_Apply_KNN.KNN/*:KNN_GetBackgroundImage_KNN.KNN/0:KNN_GetBackgroundImage_KNN.KNN/1:MOG2_Apply_Mog2.Mog2/*:MOG2_GetBackgroundImage_Mog2.Mog2/0:MOG2_GetBackgroundImage_Mog2.Mog2/1'
+
+jobs:
+  BuildAndTest:
+    runs-on: opencv-cn-mac-arm64
+    defaults:
+      run:
+        shell: bash
+    steps:
+    - name: Setup infra environment
+      if: ${{ github.event.repository.name == 'ci-gha-workflow' }}
+      run: echo "TARGET_BRANCH_NAME=3.4" >> $GITHUB_ENV
+    - name: PR info
+      run: |
+        echo "PR Author: ${{ env.PR_AUTHOR }}"
+        echo "PR Author fork: ${{ env.PR_AUTHOR_FORK }}"
+        echo "Source branch name: ${{ env.SOURCE_BRANCH_NAME }}"
+        echo "Target branch name: ${{ env.TARGET_BRANCH_NAME }}"
+    - name: Clean
+      run: find . -mindepth 1 -delete
+      working-directory: ${{ github.workspace }}
+    - name: Fetch opencv_contrib
+      run: git clone --branch ${{ env.TARGET_BRANCH_NAME }} --reference $GIT_CACHE_PATH/opencv_contrib.git https://github.com/opencv/opencv_contrib.git
```

Comment:
Let's use `$GIT_CACHE` variable for all workflows. I used `$GIT_CACHE_PATH` only for Git cache updates only in case of an issue with paths and scripts on Windows machine, that's why I created this one variable for it.

Btw, these variables have the same output.

Reply:
Okay, they are all replaced with `$GIT_CACHE` in the latest commit.

Reply:
Thanks!